jaxb-ri/txw/compiler/src/main/java/com/sun/tools/txw2/TxwOptions.java

```diff
@@ -58,4 +68,145 @@
      * {@link XmlNamespace} annotation.
      */
     public boolean noPackageNamespace;
+
+    public Language language;
+
+    public enum Language {
+        XML,
+        XMLSCHEMA,
+        COMPACT
+    }
```

Comment:
There is missing documentation on public field and also on public enum with all of its elements.

Reply:
added